src/extensions/default/MDNDocs/README.md

```diff
@@ -1,5 +1,5 @@
 # Updating the Docs
 Use the Node script [update-wpd-docs](https://github.com/MarcelGerber/update-wpd-docs) to update the `css.json` contents:
```

Comment:
If we are using MDN we should also use https://github.com/mozilla/brackets/tree/master/src/extensions/extra/MDNDocs/tools/scrape-mdn instead of @MarcelGerber tool.